test/unit/ngsi20/general/startup-test.js

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-iotagent-lib
+ *
+ * fiware-iotagent-lib is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * fiware-iotagent-lib is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with fiware-iotagent-lib.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[contacto@tid.es]
```

Comment:
contacto@tid.es -> iot_support at tid dot es

Reply:
I have not modified this because to be consistent with the headers of NGSIv1 tests. For instance: https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/unit/lazyAndCommands/active-devices-attribute-update-test.js

Reply:
Ok, `contacto@tid.es` is fine. Probably we would do an overall PR to adjust email contact in all the headers at some moment.

NTC with regards this PR